### PR TITLE
update dependencies and fix lint error

### PR DESCRIPTION
**Pull request by neighborhood999**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint global-require: 0*/
```
Comment (chentsulin) [inline src/index.js]: I prefer to keep a space before `*/`

File: src/store/configureStore.dev.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint global-require: 0*/
```
Comment (chentsulin) [inline src/store/configureStore.dev.js]: I prefer to keep a space before `*/`

File: src/store/configureStore.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* eslint global-require: 0*/
```
Comment (chentsulin) [inline src/store/configureStore.js]: I prefer to keep a space before `*/`

Comment (neighborhood999) [inline src/store/configureStore.js]: already fix.

File: .travis.yml [hunk 4]
```diff
@@ -4,5 +4,6 @@ node_js:
   - '4'
   - '5'
   - '6'
+before_script: "npm install eslint-plugin-import --save-dev"
```
Comment (chentsulin) [inline .travis.yml]: Why not install this one directly from package.json?

Comment (neighborhood999) [inline .travis.yml]: sorry, my bad.

**Pull request merged**